src/springfield/fields.py

```diff
@@ -390,12 +390,26 @@ def __init__(self, entity, *args, **kwargs):
                        Use 'self' to use the :class:`Entity` class that
                        this field is already bound to.
         """
-        self.type = entity
+        self._type = entity
         super(EntityField, self).__init__(*args, **kwargs)
 
+    @property
+    def type(self):
+        if not callable(self._type):
```

Comment:
It seems like you could if you really wanted to specify a callable here instead of an actual Entity class, I don't know why you'd want to or why you shouldn't. I poked at it with this and saw that it worked:

```
class Foo(Entity):
    bar = fields.EntityField(int)

x = Foo(bar=5)
```
I assume that's not really supported.

Also if checking type I think in reality you'd want:

```
isinstance(self._type, basestring)
```

or

```
isinstance(self._type, (str, unicode))
```

to be more explicit. Whatever you want I'll change it to.